src/main/scala/org/broadinstitute/dsde/firecloud/service/FireCloudDirectives.scala

```diff
@@ -36,19 +36,18 @@ trait FireCloudDirectives extends spray.routing.Directives with PerRequestCreato
   def encodeUri(path: String): String = FireCloudDirectiveUtils.encodeUri(path)
 
   private def generateExternalHttpPerRequestForMethod(uri: Uri, inMethod: HttpMethod) = {
```

Comment:
currently, there is a bug in here somewhere that can cause an akka-http EntitySource to be materialized twice, which results in an exception being thrown pretty much any time the passthrough directive is used. i haven't successfully tracked it down but maybe i just have fatigue from staring at this code for far too long and it'll be obvious to fresh eyes

Reply:
found where it was being consumed twice.  the `logRequestResult` directive which is wrapped around every route in `FireCloudApiService` was grabbing the entity to log it, which was throwing this exception. i've removed the use of that logging directive for now. i don't think it's critical for the first round of review and i'll figure out what's going on for followup review

Reply:
I've fixed up `logRequestResult`!